api/types/integration.go

```diff
@@ -230,6 +267,49 @@ func (s *IntegrationSpecV1_AzureOIDC) Validate() error {
 	return nil
 }
 
+// CheckAndSetDefaults validates the configuration for GitHub integration subkind.
+func (s *IntegrationSpecV1_GitHub) CheckAndSetDefaults() error {
+	if s == nil || s.GitHub == nil {
+		return trace.BadParameter("github is required for %q subkind", IntegrationSubKindGitHub)
+	}
+	if s.GitHub.Organization == "" {
```

Comment:
Can we validate organization name?

Reply:
added some validation in `api/types/integration_github.go`